core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1926,9 +1936,18 @@ void onTerminated(InternalSubchannel is) {
 
         @Override
         void onStateChange(InternalSubchannel is, ConnectivityStateInfo newState) {
-          handleInternalSubchannelState(newState);
```

Comment:
I'd have expected the handleInternalSubchannelState() method to be deleted. But it looks like it is used by OOB channel. We will probably want to resolve that as well at some point to make it explicit, but for now we can just put a TODO where handleInternalSubchannelState is called.

Reply:
ACK.